crates/biome_fs/src/fs/os.rs

```diff
@@ -244,9 +244,21 @@ fn handle_dir_entry<'scope>(
             Some(
                 "package.json"
                     | "package-lock.json"
+                    | "package-lock.yaml"
+                    | ".npmrc"
+                    | "npm-shrinkwrap.json"
+                    | "yarn.lock"
+                    | "pnpm-lock.yaml"
+                    | "pnpm-workspace.yaml"
+                    | "composer.json"
+                    | "composer.lock"
                     | "typescript.json"
                     | "tsconfig.json"
                     | "jsconfig.json"
+                    | "bun.lockb"
+                    | "bunfig.toml"
```

Comment:
```suggestion
                    | "npm-shrinkwrap.json"
                    | "yarn.lock"
                    | "composer.json"
                    | "composer.lock"
                    | "typescript.json"
                    | "tsconfig.json"
                    | "jsconfig.json"
```

Reply:
My bad :( 
The file change wasn't included in the commit